crates/ty_python_semantic/src/types.rs

```diff
@@ -9197,6 +9234,56 @@ impl<'db> BoundSuperType<'db> {
     }
 }
 
+#[salsa::interned(debug)]
+pub struct TypeIsType<'db> {
+    #[return_ref]
```

Comment:
I don't think we need/want a `return_ref` on a field that is a `Type` -- `Type` is small and `Copy`